docs/enhanced-depends-logic.md

```diff
@@ -25,7 +25,10 @@ available task results is as follows:
 | `.Omitted` | Task Omitted | Task was omitted |
 | `.Daemoned` | Task is Daemoned and is not Pending | |
 
-For convenience, if an omitted task result is equivalent to `(task.Succeeded || task.Skipped || task.Daemoned)`.
+A tasks is considered `Skipped` if its `when` condition evaluates to false. On the other hand, if a task doesn't run
+because its `depends` evaluated to false it is `Omitted`.
+
+For convenience, an omitted task result is equivalent to `(task.Succeeded || task.Skipped || task.Daemoned)`.
 
 For example:
```

Comment:
You didn't change this line much (just removed the grammatically incorrect "if"), but the word "omitted" here is very confusing given that it means the literal "omitted" and not the term `.Omitted` defined above.

We should use a different word:
```suggestion
An unspecified result is equivalent to `(task.Succeeded || task.Skipped || task.Daemoned)`. For example:
```

And should have less space between it and the example, which also alleviates confusion

Reply:
Completed in #13110